src/paper/utils.py

```diff
@@ -214,6 +215,23 @@ def get_cache_key(obj_type, pk):
     return f"{obj_type}_{pk}"
 
 
+def convert_to_rgb(pil_image: Image.Image) -> Image.Image:
+    """
+    Convert PIL image to RGB format, handling RGBA, LA, and P modes.
+
+    Returns:
+        PIL Image object in RGB mode
+    """
+    if pil_image.mode in ("RGBA", "LA", "P"):
+        background = Image.new("RGB", pil_image.size, (255, 255, 255))
+        if pil_image.mode == "P":
+            pil_image = pil_image.convert("RGBA")
+        mask = pil_image.split()[-1] if pil_image.mode in ("RGBA", "LA") else None
+        background.paste(pil_image, mask=mask)
+        pil_image = background
+    return pil_image
```

Comment:
This is too specific to include in a generic utility file. Could you add it to the service that uses it (and add a test)?